sdk/loadtestservice/azure-developer-loadtesting/CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # Release History
 
+## 1.0.0b3 (2023-01-01)
+
+### Features Added 
+- Method added `azure.developer.loadtesting.LoadTestRunClient.list_metric_namespaces`
+- Method added `azure.developer.loadtesting.LoadTestRunClient.list_metric_definitions`
+- Method added `azure.developer.loadtesting.LoadTestRunClient.list_metrics`
+- Method added `azure.developer.loadtesting.LoadTestRunClient.create_or_update_app_components`
+- Method added `azure.developer.loadtesting.LoadTestRunClient.list_app_components`
```

Comment:
were we supposed to change app_components also to get_app_components since its dictionary and not an array

Reply:
Thanks for pointing this out! This should have been `get_app_components`, updating in ChangeLog